gazebo/rendering/Visual.cc

```diff
@@ -817,9 +817,11 @@ void Visual::SetScale(const ignition::math::Vector3d &_scale)
   {
     this->dataPtr->sceneNode->setScale(
         Conversions::Convert(this->dataPtr->scale));
-  } else {
-    gzerr << Name() << ": Size of the collision contains one or several zeros." <<
-      " Collisions may not visualize properly." << std::endl;
+  }
```

Comment:
This change is not related with the PR title. I introduced this, I can open a PR to fix it.

Reply:
done, reverted these changes